.travis.yml

```diff
@@ -21,55 +21,46 @@ env:
     - RUST_NIGHTLY_VERSION=nightly-2018-06-29
     - RUST_CLIPPY_VERSION=0.0.211
     - EJB_RUST_BUILD_DIR="$TRAVIS_BUILD_DIR/exonum-java-binding-core/rust/"
+    - SODIUM_BUILD=1
+    - ROCKSDB_BUILD=1
+    - SNAPPY_BUILD=1
+    - SKIP_COMPILE=true
```

Comment:
I'd keep that _explicit_ argument as is, because you see it each time you invoke the script from your shell history, and you'll likely see it is wrong.